supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/io/package.scala

```diff
@@ -29,7 +29,7 @@ package object io {
   }
 
   private [jes] def isFatalJesException(t: Throwable): Boolean = t match {
-    case e: HttpResponseException if e.getStatusCode == 403 => true
+    case e: HttpResponseException if e.getStatusCode == 403 || e.getStatusCode == 400 => true
```

Comment:
is it safe to generally treat 400 as a fatal JES exception?  or perhaps only for localization / delocalization stuff?